src/core/controls/DropDownList.js

```diff
@@ -0,0 +1,432 @@
+    wrapper.interactive = true;
+    wrapper.buttonMode = true;
+    wrapper.defaultCursor = 'pointer';
+
+    this.labelText = new PIXI.Text('Label', Object.assign({}, this.theme.labelStyle || {font:'15px Arial', fill : 0xDDDDDD}));
+    this.labelText.y = 0;
+    this.labelText.x = 0;
+
+    if(this.showDropDown){
+        this.labelText.style.fill = this.theme.labelTextColors ? this.theme.labelTextColors.active : '#FF93A7';
+    }else{
+        this.labelText.style.fill = this.theme.labelTextColors ? this.theme.labelTextColors.normal : '#DDDDDD';
+    }
+
+    var mark = new PIXI.Graphics();
+    mark.beginFill(0x000000);
+    mark.lineStyle(3, 0x000000);
+    mark.moveTo(0,0);
+    mark.lineTo(3,0);
+    mark.lineTo(1.5,1.5);
+    mark.lineTo(0,0);
+    mark.endFill();
+    mark.x = 215;
+    mark.y = 30;
+
+    var line = new PIXI.Graphics();
+    line.beginFill( this.theme.line  ? this.theme.line.lineColor : 0xf1f2f3);
+    line.drawRect(0, 0, this.theme.line  ? this.theme.line.width : 222, this.theme.line  ? this.theme.line.height : 2);
+    line.x = 2;
+    line.y = 60;
+    line.endFill();
+
+
+    wrapper.addChild(this.labelText);
+    wrapper.addChild(mark);
+    wrapper.addChild(line);
+
+
+
+    this.selectedItemText = new PIXI.Text(this._label, Object.assign({}, this.theme.textStyle || {font:'20px Arial', fill : 0x4E5769}));
+    this.selectedItemText.x = 0;
+    this.selectedItemText.y = 25;
+
+    wrapper.addChild(this.selectedItemText);
+
+    this.addChild(wrapper);
+
+    this.updateLabel = false;
+};
+
+/**
+ * create/update DropDownList
+ *
+ * @method createDropDown
+ */
+DropDownList.prototype.createDropDown = function () { //TODO refactoring add constans
+    if(this.elementList) {
+        if(this.showDropDown){
+            var wrapper = new PIXI.Graphics();
+            wrapper.beginFill(this.theme.background ?this.theme.background.color : 0xFFFFFF);
+            wrapper.y = 20;
+            wrapper.moveTo(0,0);
+            wrapper.lineTo(0, 43 + this.elementList.length * 40 );
+            wrapper.lineTo(240, 43 + this.elementList.length * 40 );
+            wrapper.lineTo(240, 0);
+            wrapper.lineTo(0, 0);
+            wrapper.endFill();
+
+            var border = this.theme.getImage('text-input-background-disabled-skin');
+            border = border();
+            border.x = -1;
+            border.y = 0;
+
+            border.scale.x= 1.57;
+            border.scale.y= 2.45 * (this.elementList.length - 0.2);
+
+
+            var inner = new PIXI.Container();
+            inner.y = 40;
+
+            this.elementList.forEach(function (el, i) {
+
+                var itemText = new PIXI.Text(el.text, Object.assign({}, this.theme.textStyle || {font:'20px Arial', fill : 0x4E5769})); // use own styles
+
+
+                if(typeof this.hoveredElementIndex === 'number' && this.hoveredElementIndex === i){
+                    var background = new PIXI.Graphics();
```

Comment:
The PickerList-implementation on dev uses the selected item of the list to highlight the selected item. I don't see the need for own code-highlight handling in DropDown-component itself.